documentation/jetty/modules/operations-guide/pages/deploy/index.adoc

```diff
@@ -13,108 +13,258 @@
 
 = Web Application Deployment
 
-The Jetty Web Application Deployment mechanism provides for:
+The Jetty server deploys each web application to a specific _environment_.
```

Comment:
```suggestion
Jetty deploys each web application to a specific _environment_.
```

Reply:
Ah, no. "Jetty" is not defined (typically refers to the project), we have client and server, so it's really "The Jetty server".